s4e-web/cypress/integration/auth/auth-login.spec.ts

```diff
@@ -55,6 +55,11 @@ describe('Auth', () => {
       Login
         .fillForm({ ...this.userToRegister, email: 'wrongFormat' })
         .errorsCountShouldBe(1);
-    })
+    });
+
+    it('should not allow login page for user with session', function () {
+      Login.loginAs(this.superAdmin)
+      cy.visit('/login')
```

Comment:
Shouldn't be checked if user is redirected to `/map/products`?

Reply:
Right, I'll fix it